src/app/services/terms/terms.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+@Injectable({
+  providedIn: 'root'
+})
+export class TermsService {
+
+  constructor(private http: HttpClient) {
+
+  }
+
+  getTerms(): Observable<any> {
```

Comment:
Could you specify the type that the observable returns so that the service is tied to the specified type, reduce the usage of `any`

Reply:
Hey @malep2007, I have statically typed this to a Term Model.